A6-client-retries.md

```diff
@@ -262,6 +274,10 @@ Similarly, if the RPC reaches the server, but has never been seen by the server
 
 Since retry throttling is designed to prevent server application overload, and these transparent retries do not make it to the server application layer, they do not count as failures when deciding whether to throttle retry attempts.
 
+![State Diagram](A6_graphics/transparent.png)
```

Comment:
Diagram caption changes:

s/Failure occurs in the before/Failure occurs before/
s/max_attempts/maxRetryAttempts/

Reply:
Done